CMakeLists.txt

```diff
@@ -187,6 +187,7 @@ add_project_dependency(Eigen3 REQUIRED PKG_CONFIG_REQUIRES "eigen3 >= 3.0.5")
 if(BUILD_WITH_URDF_SUPPORT)
   add_project_dependency(urdfdom_headers REQUIRED)
   add_project_dependency(urdfdom REQUIRED PKG_CONFIG_REQUIRES "urdfdom >= 0.2.0")
+  add_project_dependency(console_bridge REQUIRED)
```

Comment:
This add_project_dependency is already added [line 272](https://github.com/stack-of-tasks/pinocchio/pull/2312/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR273).

As far as I know, console_bridge is only used in the python binding.

Reply:
Well spotted, thanks !